src/opt_handlers.c

```diff
@@ -676,6 +678,14 @@ options[] = {
 	  OPT_STRLIST, 0, NULL, &trashdir_handler, NULL,
 	  { .init = &init_trashdir },
 	},
+	{ "trash_filefmt_zero", "", "file name when moved to the trash directory and no other files exist with the same name",
```

Comment:
Neither Vim nor Vifm has options with underscores, I'd prefer to keep it that way for consistency.